tl-tl/sc/getting-started-java.md

```diff
@@ -2,98 +2,98 @@
+Ang mga smart na kontrata ay sinusulat gamit ang mga mataas na lebel na lengguwahe kagaya ng Java, C#, Python, Kotlin (at iba pa...) at kinukompayl para maging AVM (ang Virtual Machine bytecode ng Neo) para mapatakbo ito sa Neo network.
 
-We currently recommend C# for developing smart contracts. The Java compiler is still in development but the current version (neoj) can handle basic methods.
+Kami ay kasalukuyang nagrerekomenda na gumamit ng C# para gumawa ng mga smart na kontrata. Ang Java compiler ay ginagawa pa rin hanggang sa ngayon pero ang kasalukuyang bersyon (neoj) ay maari nang makapagtakbo ng mga basic na method.
 
-This section contains a tutorial that guides you in configuring the Java development environment for NEO smart contracts. It also gives you an idea of ​​how to create a smart contract project and how to compile it.
+Ang seksyon na ito ay naglalaman ng isang tutoryal na gagabay sa iyo sa pagkonpigyura ng Java development environment para sa mga NEO smart na kontrata. Ito rin ay magbibigay ng ideya kong papano gumawa ng isang smart na kontratang proyekto at paano ito i-kompayl.
 
-Notes: The process involves the following steps:
-1. Write Java code (.java) for classes that extend FunctionCode or VerificationCode which is part of the Neo Framework Library (JAR)
-2. Use the normal Java compiler to compile code into Java bytecode (.class)
-3. Build the neoj (C#) compiler that converts JVM code into AVM code (neoj.exe on Windows)
-4. Use neoj to compile your .class file (.avm)
-5. Download Neo's Node GUI to connect to Neo's Testnet network
-6. Deploy your .avm script to publish the smart contract to the network
-7. Invoke your .avm script to execute your smart contract
+Mga Paalala: Ang proseso ay naglalaman ng mga sumusunod na mga hakbang:
+1. Sumulat ng Java na code (.java) para sa mga class na umi-extend sa FunctionCode o VerificationCode na parte sa Neo Framework Library (JAR)
+2. Gumamit ng normal na Java compiler para i-kompayl ang code para maging Java bytecode (.class)
+3. I-build ang neoj (C#) na compiler na kino-convert ang JVM na code para maging AVM na code (neoj.exe sa Windows)
+4. Gamitin ang neoj para icompile ang iyong .class na file (.avm)
+5. I-downloand ang Node GUI ng Neo para kumonekta sa Testnet na network ng Neo.
+6. I-deploy ang iyong .avm na script para ipublish ang smart na kontrata sa network
+7. I-invoke ang iyong .avm na script para patakbuhin ang iyong smart na kontrata
 
-### Detailed Instructions
+### Mga Detalyadong Instruksyon
 
-## Tools
+## Mga Tool
 
-The most efficient way of getting these steps done is to download and compile all the tools you will need:
+Ang pinakaepisyenteng pamamaraan para magawa ang mga sumusunod na mga hakbang ay ang pagdownload at pagkompayl ng lahat ng tools na kakailanganin mo:
```

Comment:
you could translate `tools` to `kasangkapan` or `kagamitan`

Reply:
Changed `tool` to `kagamitan`